contracts/SpokePool.sol

```diff
@@ -78,7 +76,7 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
      ****************************************/
     event SetXDomainAdmin(address indexed newAdmin);
     event SetHubPool(address indexed newHubPool);
-    event EnabledDepositRoute(address indexed originToken, uint256 indexed destinationChainId, bool enabled);
+    event EnabledDepositRoute(uint256 indexed destinationChainId, address indexed originToken, bool enabled);
```

Comment:
I think this packs variables better? or maybe it doesn't matter for events?

Reply:
dont think it makes a big diff